docs/getting-started/roles/index.rst

```diff
@@ -0,0 +1,67 @@
+==================================
+Built-in Role-Based Access Control
+==================================
+
+oso includes a library that makes it easy to add roles to your application.
```

Comment:
Would suggest reducing the subjective so much here. I.e. remove the "easy"s.

oso includes support for adding roles directly to your application via our ORM integrations.
...
We generate Polar rules that you use in your policies to write rules based on the roles instead of directly over your users.

Reply:
Edited.